compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/window/ComposeSceneMediator.uikit.kt

```diff
@@ -118,12 +124,51 @@ internal class ComposeSceneMediator(
         )
     }
 
+    private val semanticsOwnerListener: PlatformContext.SemanticsOwnerListener by lazy {
```

Comment:
We don't need to store the reference to it in mediator. Just create `private inner class` and call create object for platform context.

Reply:
I'd leave it for future refactor sweep, where we track, what's lazy, what's reused, initialisation order, etc.